ada/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+
+class RedirectUrl(models.Model):
+
+    url = models.URLField(
+        max_length=255,
+        blank=True,
+        null=True,
+        default="https://hub.momconnect.za/confirmredirect",
+    )
+    content = models.TextField(default="This entry has no copy")
+    refresh_url = models.CharField(
+        max_length=200, null=False, blank=False, default="https://hub.momconnect.za"
+    )
+    symptom_check_url = models.CharField(
+        max_length=200, null=False, blank=False, default="http://symptomcheck.co.za"
+    )
```

Comment:
This is the URL that we're redirecting to. Why are we using `symptom_check_url`, a `CharField`, instead of `url`, a `UrlField`? It seems like `url` is not being used at all